cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -2,11 +2,12 @@ name "Azure Databricks Rightsize Compute Instances"
 rs_pt_ver 20180301
 type "policy"
 short_description "Checks for instances that have inefficient utilization for a certain lookback period (in days). \n See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/databricks/rightsize_compute/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
 severity "low"
 category "Cost"
 default_frequency "weekly"
 info(
-  version: "0.2.1",
+  version: "0.3.0",
   provider: "Azure",
   service: "Databricks",
   policy_set: "Databricks",
```

Comment:
Line 21 - please add `category` filed to `"param_email"` parameter. See other policies for examples